### PR TITLE
Enabling audiences UI

### PR DESCRIPTION
**Pull request by davimacedo**
See https://github.com/parse-community/parse-server/pull/3861

### CODE REVIEW

**Comment by natanrolnik** [comment]
This is a great feature to have, thanks for the contribution, @davimacedo!
@flovilmart, could you review this and the PR for the server?

**Comment by flovilmart** [comment]
Thanks @davimacedo for the feature, can you address the lint errors. Also, I'd prefer we kept the method calls on one line, this is hardly readable that way for [ex](https://github.com/parse-community/parse-dashboard/pull/712/files#diff-1f6d18953ffbb2f56a74218c4e0df9c1R33):

```
const path = action.limit ? `push_audiences?audience_limit=${action.limit}` : 'push_audiences';
const options = {useMasterKey: true}
let promise = action.app.apiRequest('GET', path, {}, options)
```

**Comment by davimacedo** [comment]
Done.

File: Parse-Dashboard/parse-dashboard-config.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "apps": [{
-    "serverURL": "",
-    "appId": "",
-    "masterKey": "",
-    "appName": "",
+    "serverURL": "https://parseapi.back4app.com",
```
Comment (natanrolnik) [inline Parse-Dashboard/parse-dashboard-config.json]: Ooops :)

**Comment by natanrolnik** [comment]
Please fix the config.json and we will merge it!

File: Parse-Dashboard/parse-dashboard-config.json (hunk 1 above)
Comment (davimacedo) [inline Parse-Dashboard/parse-dashboard-config.json]: Sorry... at least it was a test app :)

**Comment by davimacedo** [comment]
done

**Pull request merged**